### PR TITLE
Refresh script (fixes #88)

### LINKED ISSUE #88: EVE doesn't make key masks immutable; recheck masks periodically
API key masks and types need to be periodically checked in case someone submits a full API key then removes permissions from it.

### PR DESCRIPTION
**Pull request by jboning**

### CODE REVIEW

**Comment by Acidity** [comment]
When trying to run the refresh script manually, I'm getting an ImportError, is there perhaps some additional setup required for this?

(brave)tyler@debianVM:~/brave/core$ python brave/core/scripts/refresh.py 
Traceback (most recent call last):
  File "brave/core/scripts/refresh.py", line 4, in <module>
    from brave.core import core_loadapp
ImportError: No module named core

**Comment by jboning** [comment]
Do you have a virtualenv set up for core? Make sure you're running the right python. Though that wouldn't seem to lead to this. Also, have you done "python setup.py develop" or "pip install -e ." (I think that is the invocation)? That _might_ make a difference.

**Comment by Acidity** [comment]
In a virtualenv (hence the (brave) next to the prompt, and I just ran both commands you suggested, and still getting the error, same with your case migration script. Might be just my set up though.

**Comment by jboning** [comment]
Oh, derp. I've gotten used to running ../env/python manually. Weird. I think I saw a problem like this once, but I don't remember anything I did to make it go away.

**Comment by Acidity** [comment]
I get the same problem with trying to run case_migration.py as well. I'll try this again tomorrow, though ideally we'd be able to get someone else to test and see if they have the same problem or not.

**Comment by Acidity** [comment]
Yeah... problem was I wasn't running it in paster.

File: brave/core/scripts/refresh.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+from brave.core import core_loadapp
+if __name__ == "__main__":
+    core_loadapp("config:"+sys.argv[1] if len(sys.argv) > 1 else None)
+
+import time
+
+from requests.exceptions import HTTPError
+
+from brave.core.key.model import EVECredential
+from brave.core.util.signal import validate_key
```
Comment (Acidity) [inline brave/core/scripts/refresh.py]: Validate_key got deleted a while back, might want to fix this (replaced by pull iirc).

Comment (jboning) [inline brave/core/scripts/refresh.py]: Whoops, sure enough. I'm really not on the ball with this one.

**Comment by jboning** [comment]
What do you mean by running in paster? You shouldn't need to invoke paster yourself at all, core_loadapp is supposed to pull in all of the paster framework stuff.

**Comment by Acidity** [comment]
Well, trying "python brave/core/scripts/refresh.py" leads to an import error, but doing "paster shell local.ini" then "import brave.core.scripts.refresh" works perfectly.

**Pull request merged**